pypodcastparser/Item.py

```diff
@@ -236,13 +232,17 @@ def set_itunes_episode(self, tag):
         """Parses the episode number and sets value"""
         try:
             self.itunes_episode = tag.string
+            if –32,767 > self.itunes_episode or self.itunes_episode > 32,767 or self.itunes_episode == None:
```

Comment:
I'd suggest having the smallint-checker be in `episodeingestion`, that way we can log the offending episodes guid, show_id, and whatever column that isn't in range before we set the value to 0.